pallets/acurast/src/migration.rs

```diff
@@ -29,29 +32,47 @@ pub mod v1 {
     }
 }
 
-pub fn migrate_to_v2<T: Config>() -> Weight {
+pub fn migrate<T: Config>() -> Weight {
+    let migrations: [&dyn Fn() -> Weight; STORAGE_VERSION_ - 1] =
```

Comment:
Should be:
```suggestion
    let onchain_version = Pallet::<T>::on_chain_storage_version();
    let migrations: [&dyn Fn() -> Weight; onchain_version] =
```

Do you agree?